plugins/linux/vm.sh

```diff
@@ -4,18 +4,18 @@ print_vm() {
     printf "%s\`%s\tL\t%s\n" $1 $2 $3
 }
 
-MEM=($(free -b | grep ^Mem:))
+MEM=($(free -k | grep ^Mem:))
 MEM_TOTAL=${MEM[1]}
 # For consistency across platforms, count cache as free, not used
 let MEM_USED=${MEM[1]}-${MEM[3]}-${MEM[5]}-${MEM[6]}
 let MEM_FREE=${MEM[3]}+${MEM[5]}+${MEM[6]}
-MEM_PERC=`echo "scale=2; $MEM_USED/$MEM_FREE" | bc`
+let MEM_PERC=100*$MEM_USED/$MEM_TOTAL
```

Comment:
MEM_PERC is a percentage... a number between 0 (0%) and 1 (100%) .  You patch changes the meaning of the returned value and will confuse people using this.